<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuildWindow</class>
 <widget class="QMainWindow" name="BuildWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Evaluate Window</string>
  </property>
  <property name="iconSize">
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="Select_TRUE_set">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>True Set</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Select_TRUE_set_sign">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select True Set</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="Select_FALSE_set">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>False Set</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Select_FALSE_set_sign">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select False Set</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Data_Import">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Import</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Model_Fit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Fit Model</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="Model_save">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>160</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Model Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_Model">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>160</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Model</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="Select">
    <property name="title">
     <string>Select Page</string>
    </property>
   </widget>
   <addaction name="Select"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
